### DISCUSSION TITLE
In production deny access by default

### DISCUSSION THREAD

**Discussion by core45** [opcodesio/log-viewer#292] [Ideas]
The package is insecure by default. If you deply an app with this package with default configuration anyone can access quite sensitive information.

This is very bad approach in my opinion.

In the production environment any access should be denied by default.

**Comment by arukompas** [comment]
hey @core45 , a very valid point! Thanks 🙏

From **v3.3.0**, Log Viewer will be unauthorized by default in production environments (`App::isProduction()`)
